doc/manual/config-basic.rst

```diff
@@ -119,7 +119,7 @@ statements that require non-standard evaluation.
 Checking your configuration
 ---------------------------
 From the front page the *Config checker* is available. This tool will
-do a basic sanity check of your DOMjudge setup and gives helpful hints
+do a basic coherence check check of your DOMjudge setup and gives helpful hints
```

Comment:
Check check. Double check

Reply:
I think `sanity check` is clearer than `coherence` and I honestly don't see the issue with the former.

Reply:
I don't object to changing wording persé, but I really think that should not be done in an automated fashion like this.
On a case by case basis, we should consider whether a replacement makes sense.

Reply:
I think we now at least have a list of all places, so we can check them.

Reply:
This was not done automated, so the only things the tool did find were the dummy and sanity which are in my opinion in the same category. Never intended in the wrong way but can be considered wrong. 

Shall I just take the part about the `sanity` out or do you also disagree with the `dummy`?

Reply:
I'd say we should just leave the upstream `config.guess` as is (or replace it with a new upstream version). The `dummy` I'm ok changing, but only if the new word makes sense. In various of the cases below `demo` doesn't make sense, for example since `dummy` represents a parameter thas is not in use (anymore). TBH as such, I don't see the issue: dummy clearly has this meaning, for example in:

- https://en.wikipedia.org/wiki/Dummy_pronoun
- https://en.wikipedia.org/wiki/Crash_test_dummy
- https://en.wikipedia.org/wiki/Dummy_round

So what I object to is words that clearly have *non-non-inclusive* meaning and replace these by words that describe more poorly the intended meaning.